tests/compute/nonuniformres-nested-rwstructuredbuf.slang

```diff
@@ -0,0 +1,31 @@
+// vk-texture-indexing.slang
+
```

Comment:
Probably you want to delete these lines.

Reply:
done